sbn.it.js

```diff
@@ -51,67 +56,130 @@ var typeMapping = {
 	// "documento cartografico manoscritto"
 };
 
-function detectWeb(doc, url) {
-	if (url.includes("full.jsp")) {
-		var type = ZU.xpathText(doc, '//tr[ td[contains(text(), "Tipo documento")] ]/td[contains(@class,"detail_value")]');
-		// Z.debug(type.trim());
-		return typeMapping[type.trim().toLowerCase()] || "book";
+// Used when "Visualizza tutti" or "Visualizza selezionati" button is pressed:
+// A single item per page but pages change dynamically with scrolling
+function getDetailedResults(doc, url, checkOnly) {
+	// Detailed results
+	var parser = new DOMParser();
+	var innerdoc = parser.parseFromString(doc.documentElement.outerHTML, "text/html");
+	var results = ZU.xpath(innerdoc, '//div[@data-to-post-item]/@data-to-post-item');
```

Comment:
We don't need DOMParser or XPath at all here, I think...

```suggestion
	var results = doc.querySelectorAll('[data-to-post-item]');
```